pages/[prefix]/drops/[id].vue

```diff
@@ -1,17 +1,15 @@
 <template>
   <UnlockableCollectionBanner :drop="drop" />
-  <CollectionDropContainer v-if="dropType === 'paid'" :drop="drop" />
+  <!-- <CollectionDropContainer v-if="dropType === 'paid'" :drop="drop" /> -->
   <CollectionDropHolderOfGenerative
-    v-else-if="dropType === 'generative' && Boolean(drop.holds)"
+    v-if="dropType === 'holder' && Boolean(drop.holder_of)"
     :drop="drop" />
   <CollectionDropPaidGenerative
-    v-else-if="dropType === 'generative' && Number(drop.price)"
+    v-else-if="dropType === 'paid' && Number(drop.price)"
```

Comment:
We do need to check `holder_of` or `price`, because we have already independent type for different drops.

Reply:
Did you mean `"we don't need to check"`?

Reply:
Yes, let remove them `&& Number(drop.price)` ` && Boolean(drop.holder_of)`